pkg/shell/terraform.go

```diff
@@ -76,14 +76,14 @@ func needsInit(tf *tfexec.Terraform) bool {
 
 func initModule(tf *tfexec.Terraform) error {
 	var err error
-	log.Printf("executing \"terraform -chdir=%s init\"\n", tf.WorkingDir())
 	if needsInit(tf) {
+		log.Printf("initializing terraform directory %s", tf.WorkingDir())
```

Comment:
You lost the new line on move. Typo or intentional?

Reply:
Unintentional but caused me to read the documentation for log more carefully for its newline behavior which I've never quite understood. It [states](https://pkg.go.dev/log#pkg-overview):

>  Every log message is output on a separate line: if the message being printed does not end in a newline, the logger will add one.

So, I'm removing all trailing `\n` from every log message in this package.